.gitmodules

```diff
@@ -5,7 +5,8 @@
 [submodule "WW3"]
   path = WW3
   url = https://github.com/NOAA-EMC/WW3
-  branch = dev/ufs-weather-model
+#  branch = dev/ufs-weather-model
```

Comment:
Ali, is the dev/ufs-weather-model sync-ed with ww3 develop branch? I thought we are stilling using dev/ufs-weather-model branch. What are the differences between dev/ufs-weather-model branch and develop branch?